identity/backend/src/main/resources/application.yaml

```diff
@@ -1,13 +1,8 @@
 spring:
```

Comment:
The settings at the "bottom" contradict with the single [application properties](https://github.com/camunda/camunda/blob/main/dist/src/main/resources/application.properties) settings, i.e., management port, context path, health endpoints, ... basically, everything under `management` and `server` must be removed.

```
management:
  server:
    port: 8082
  endpoints:
    web:
      exposure:
        include: health,prometheus
server:
  compression:
    enabled: true
  http2:
    enabled: true
  servlet:
    encoding:
      charset: UTF-8
      force: true
    context-path: ${IDENTITY_BASE_PATH:}
  max-http-request-header-size: ${SERVER_REQUEST_HEADER_MAX_SIZE:8KB}
```